examples/WebInterface/esp8266-AC-control.ino

```diff
@@ -0,0 +1,259 @@
+#include <FS.h>
+#include <ESP8266WiFi.h>
+#include <ESP8266mDNS.h>
+#include <WiFiUdp.h>
+#include <ESP8266HTTPUpdateServer.h>
+#include <ESP8266WebServer.h>
+#include <WiFiManager.h>
+#include <ArduinoJson.h>
+#include <IRremoteESP8266.h>
+#include <IRsend.h>
+
+//// ###### User configuration space for AC library classes ##########
+
+#include <ir_Coolix.h> // replace library based on your AC unit model, check https://github.com/markszabo/IRremoteESP8266
+
+#define auto_mode kCoolixAuto
+#define cool_mode kCoolixCool
+#define dry_mode kCoolixDry
+#define heat_mode kCoolixHeat
+#define fan_mode kCoolixFan
+
+#define fan_auto kCoolixFanAuto
+#define fan_min kCoolixFanMin
+#define fan_med kCoolixFanMed
+#define fan_hi kCoolixFanMax
+
+const uint16_t kIrLed = D2;  // ESP8266 GPIO pin to use. Recommended: 4 (D2).
```

Comment:
Change `D2` to just `4`. Some ESP8266 boards don't have the labels for pins like `D2`. This will make it compile on other boards easier.